### DISCUSSION TITLE
Document Request: Laravel

### DISCUSSION THREAD

**Discussion by entity** [maizzle/framework#705] [Ideas]
Hey guys,

Thanks for building such a fantastic solution to emails, as it’s often a pain especially with styling.

It would be great to see an example covering the [Laravel Framework](https://laravel.com).

I use Laravel myself, and I’ve not been sure how to integrate them together. This would help many who also use the framework, and want to easily style emails.

Thanks a ton

**Comment by cossssmin** [comment]
Yup, we definitely need a guide for this, and it’s on the list! 

Ralph’s article is a great starting point, though it doesn’t use Maizzle, but can mostly get you there:

https://ralphjsmit.com/laravel-emails-tailwind-css

In a nutshell, you could keep the Maizzle project in your Laravel app, configure it to output .blade.php files at the desired location, and finally have a build step as part of your frontend compilation that triggers a Maizzle build.

Will try to get a guide up soon 👍

**Reply by entity** [reply]
> Yup, we definitely need a guide for this, and it’s on the list!
> 
> Ralph’s article is a great starting point, though it doesn’t use Maizzle, but can mostly get you there:
> 
> https://ralphjsmit.com/laravel-emails-tailwind-css
> 
> In a nutshell, you could keep the Maizzle project in your Laravel app, configure it to output .blade.php files at the desired location, and finally have a build step as part of your frontend compilation that triggers a Maizzle build.
> 
> Will try to get a guide up soon 👍

Thanks for the reply buddy! That’s awesome. It would be great to have for sure!
